DistFiles/aboutBox.htm

```diff
@@ -1,54 +1,60 @@
-    <p><a href="http://sourceforge.net/projects/mediainfo/">MediaInfo</a> © 2002-2009 Jerome Martinez (<a href="http://www.gnu.org/licenses/old-licenses/lgpl-2.1.html">LGPL</a>)</p>
+    <p><a href="https://github.com/segmentio/Analytics.NET">Analytics.Net</a> © 2017 Segment.IO (<a href="https://opensource.org/licenses/MIT">MIT</a>)    </p>
+    <p><a href="https://github.com/autofac/Autofac">Autofac</a> © 2019 Autofac Contributors (<a href="https://opensource.org/licenses/MIT">MIT</a>)</p>
+    <p><a href="https://dotnetzip.codeplex.com/">Iconic DotNetZip</a> </p>
+    <p><a href="https://ffmpeg.org/">FFmpeg</a> © 2000-2018 The FFmpeg Developers (<a href="https://www.gnu.org/licenses/old-licenses/lgpl-2.1.html">LGPL</a>)</p>
+    <p><a href="https://sourceforge.net/projects/mediainfo/">MediaInfo</a> © 2002-2018 Jerome Martinez (<a href="https://www.gnu.org/licenses/old-licenses/lgpl-2.1.html">LGPL</a>)</p>
     <p>MediaPlayer © 2000-2010 MPlayer Team (<a href="https://gnu.org/licenses/gpl.html">GPL</a>) (SayMore does not &quot;link&quot; to MediaPlayer)</p>
     <p><a href="https://github.com/Moq/moq4/">Moq</a> Copyright (c) 2007. Clarius Consulting, Manas Technology Solutions (<a href="https://github.com/Moq/moq4/blob/master/License.txt">license</a></p>
-    <p><a href="http://naudio.codeplex.com/">NAudio</a> Mark Heath (Copyright unknown) (<a href="http://naudio.codeplex.com/license">MS-PL</a>)</p>
-    <p><a href="http://www.nunit.org">NUnit</a> Copyright © 2002-2012 Charlie Poole, Copyright © 2002-2004 James W. Newkirk, Michael C. Two, Alexei A. Vorontsov, Copyright © 2000-2002 Philip A. Craig (<a href="http://nunit.org/index.php?p=license&r=2.6.2">Nunit License</a>)</p>
-    <p><a href="https://github.com/sillsdev/libpalaso">Palaso Library</a> © <a href="http://sil.org">SIL International</a> (<a href="http://sil.mit-license.org/">MIT</a>)</p>
-    <p><a href="https://bitbucket.org/hatton/l10nsharp">L10N Sharp</a> Localization library: © <a href="http://sil.org">SIL International</a> (<a href="http://sil.mit-license.org/">MIT</a>) </p>
+    <p><a href="https://github.com/naudio/NAudio">NAudio</a> Mark Heath (Copyright unknown) (<a href="https://github.com/naudio/NAudio/blob/master/license.txt">MS-PL</a>)</p>
+    <p><a href="https://github.com/nunit/nunit">NUnit</a> Copyright © 2002-2018 Charlie Poole, Copyright © 2002-2004 James W. Newkirk, Michael C. Two, Alexei A. Vorontsov, Copyright © 2000-2002 Philip A. Craig (<a href="https://github.com/nunit/nunit/blob/master/LICENSE.txt">Nunit License</a>)</p>
+    <p><a href="https://github.com/sillsdev/libpalaso">Palaso Library</a> © <a href="https://www.sil.org/">SIL International</a> (<a href="https://sil.mit-license.org/">MIT</a>)</p>
+    <p><a href="https://github.com/sillsdev/l10nsharp">L10N Sharp</a> Localization library: © <a href="https://www.sil.org/">SIL International</a> (<a href="https://sil.mit-license.org/">MIT</a>) </p>
```

Comment:
I think we also use:

`<p><a href="https://www.newtonsoft.com/json">Newtonsoft.Json</a> Copyright © 2018 James Newton-King (<a href="https://www.nuget.org/packages/Newtonsoft.Json/12.0.1/license">MIT</a>)</p>`

Reply:
Done.